src/platform/qt/SettingsView.cpp

```diff
@@ -322,6 +322,7 @@ void SettingsView::reloadConfig() {
 	loadSetting("savestatePath", m_ui.savestatePath);
 	loadSetting("screenshotPath", m_ui.screenshotPath);
 	loadSetting("patchPath", m_ui.patchPath);
+	m_ui.libraryStyle->setCurrentIndex(loadSetting("libraryStyle").toInt());
```

Comment:
Please move this to the bottom of the function (or at least outside of this block).